src/Worker/CheckRelMeProfileLink.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2022, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Worker;
+
+use DOMDocument;
+use Friendica\DI;
+use Friendica\Core\Logger;
+use Friendica\Model\Profile;
+use Friendica\Model\User;
+use Friendica\Network\HTTPClient\Client\HttpClientAccept;
+use Friendica\Network\HTTPClient\Client\HttpClientOptions;
+use Friendica\Util\Network;
+use Friendica\Util\Strings;
+
+/* This class is used to verify the homepage link of a user profile.
+ * To do so, we look for rel="me" links in the given homepage, if one
+ * of them points to the Friendica profile of the user, a verification
+ * mark is added to the link.
+ *
+ * To reverse the process, if a homepage link is given, it is displayed
+ * with the rel="me" attribute as well, so that 3rd party tools can
+ * verify the connection between the two pages.
+ *
+ * This task will be performed by the worker on a daily basis _and_ every
+ * time the user changes their homepage link. In the first case the priority
+ * of the task is set to LOW, with the second case it is MEDIUM.
+ *
+ * rel-me microformat docs https://microformats.org/wiki/rel-me
+ */
+class CheckRelMeProfileLink
+{
+	/* Cheks the homepage of a profile for a rel-me link back to the user profile
+	 *
+	 * @param $uid (int) the UID of the user
+	 */
+	public static function execute(int $uid)
+	{
+		Logger::notice('Verifying the homepage', ['uid' => $uid]);
+		Profile::update(['homepage_verified' => false], $uid);
+		$homepageUrlVerified = false;
+		$owner = User::getOwnerDataById($uid);
+		if (!empty($owner['homepage'])) {
+			$xrd_timeout = DI::config()->get('system', 'xrd_timeout' ?? 20);
```

Comment:
```suggestion
			$xrd_timeout = DI::config()->get('system', 'xrd_timeout') ?? 20;
```

Or as alternative to @annando , plz beware of the wrong syntax :-)